cypress/integration/articleFlows/hideArticleComments.spec.js

```diff
@@ -0,0 +1,89 @@
+describe('Hiding article comments flow', () => {
```

Comment:
I think we need to rework this test a little bit to avoid logging in/out as two different users. I totally understand why it's written this way, but something we've noticed internally is that we see a lot of flakiness in specs which sign in/out as different users 🙈 

In `seeds_e2e.rb` we have a test article seeded before the tests start, and it has a single comment on it (created by the admin user). I think to get around the sign in/out issue, we could split this into two tests:

**The admin user hides and unhides the comment on the Test Article:**

- Log in as the admin user
- Visit the "Test article" - you can do this by visiting the home page then: 
```js
cy.findAllByRole('link', { name: 'Test article' }).first().click({ force: true })`
```
- Then carry on with hiding/unhiding the comment, checking the UI for the admin user only

**A different user views an article with comments already hidden:**

- For this I think we'd need to add to the `seeds_e2e.rb` to make sure a hidden comment is added to the Test article, so when the test executes you only need to login as one of the test users and view the hidden comment UI

Let me know if you need a hand with this - it's definitely tricky testing these areas that rely on multiple users interacting!

Reply:
@aitchiss Thank you. Let me give this splitting approach a shot and push changes. Will reach out if I need help.